src/interest.sol

```diff
@@ -0,0 +1,42 @@
+// Copyright (C) 2018 Centrifuge
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity >=0.5.12;
+
+import "./math.sol";
+
+contract Interest {
+    // compounding takes in chi (accumulated rate), speed (accumulation per second), rho (when the rate was last updated),
+    // and pie (total debt of all loans with one rate divided by that rate).
```

Comment:
I would not mention loans and debt here. It should be independent from our usage with loans and debt. 

I also have a different understanding of rate. For me rate != chi

Reply:
how would you explain pie without using `debt`?

Reply:
if it is a math/interest repo it might be used in contexts where the debt is actually a savings/deposit